docs/4.0/components/breadcrumb.md

```diff
@@ -32,3 +32,22 @@ Similar to our navigation components, breadcrumbs work fine with or without the
   <span class="breadcrumb-item active">Bootstrap</span>
 </nav>
 {% endexample %}
+
+
+## Regarding accessibility
+
+If you’re using breadcrumbs to provide a navigation, be sure to add a `aria-label="breadcrumb"` to describe the type of navigation provided in the `<nav>` element.
+
+Also apply an `aria-current="page"` to the last `<a>` of the set to indicate that it represents the current page.
+
+More information on [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices/#breadcrumb).
```

Comment:
Feel like the above three lines should be a single paragraph vs three separate lines.